src/openvic-simulation/dataloader/NodeTools.cpp

```diff
@@ -356,3 +406,7 @@ std::ostream& OpenVic::operator<<(std::ostream& stream, name_list_t const& name_
 callback_t<std::string_view> NodeTools::assign_variable_callback_string(std::string& var) {
 	return assign_variable_callback_cast<std::string_view>(var);
 }
+
+callback_t<std::string_view> NodeTools::assign_variable_callback_string(std::string_view& var) {
+	return assign_variable_callback_cast<std::string_view>(var);
+}
```

Comment:
`assign_variable_callback` works for this case, the other `assign_variable_callback_string` function was only needed due to the mismatch in parameters (`std::string_view` vs `std::string`).